message_parser/src/main/java/cumulus_message_adapter/message_parser/JsonUtilities.java

```diff
@@ -0,0 +1,37 @@
+package cumulus_message_adapter.message_parser;
+
+import java.util.Map;
+
+import java.lang.reflect.Type;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+/**
+ * Utilities for converting JSON to string/map
+ */
+public class JsonUtilities {
+
```

Comment:
To prevent instantiation, add a `private` no-arg constructor, which is standard practice for "utility" classes that consist of only static methods. This makes it clear that nobody should ever create an instance of this class.